app/javascript/test/Jobs.test.tsx

```diff
@@ -22,117 +41,284 @@ Object.defineProperty(window, 'matchMedia', {
-        positionLocation: 'Multiple Locations',
-        organizationName: 'Office of Acquisition and Logistics 2',
-        minimumPay: 122000,
-        maximumPay: 150000,
-        rateIntervalCode: 'PA',
-        applicationCloseDate: 'September 21, 2023'
-      },
-      {
-        positionTitle: 'Contract Specialist 3',
-        positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
-        positionLocation: 'Multiple Locations',
-        organizationName: 'Office of Acquisition and Logistics 3',
-        minimumPay: 122000,
-        maximumPay: 150000,
-        rateIntervalCode: 'PA',
-        applicationCloseDate: 'September 22, 2023'
-      },
-      {
-        positionTitle: 'Contract Specialist 4',
-        positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
-        positionLocation: 'Multiple Locations',
-        organizationName: 'Office of Acquisition and Logistics 4',
-        minimumPay: 122000,
-        maximumPay: 150000,
-        rateIntervalCode: 'PA',
-        applicationCloseDate: 'September 23, 2023'
-      },
-      {
-        positionTitle: 'Contract Specialist 5',
-        positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
-        positionLocation: 'Multiple Locations',
-        organizationName: 'Office of Acquisition and Logistics 5',
-        minimumPay: 122000,
-        maximumPay: 150000,
-        rateIntervalCode: 'Per Year',
-        applicationCloseDate: 'September 24, 2023'
-      },
-      {
-        positionTitle: 'Contract Specialist 6',
-        positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
-        positionLocation: 'Multiple Locations',
-        organizationName: 'Office of Acquisition and Logistics 5',
-        minimumPay: 122000,
-        maximumPay: 150000,
-        rateIntervalCode: 'Per Hour',
-        applicationCloseDate: 'September 24, 2023'
-      },
-      {
-        positionTitle: 'Contract Specialist 7',
-        positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
-        positionLocation: 'Multiple Locations',
-        organizationName: 'Office of Acquisition and Logistics 5',
-        minimumPay: 122000,
-        maximumPay: 150000,
-        rateIntervalCode: 'Without Compensation',
-        applicationCloseDate: 'September 24, 2023'
-      },
-      {
-        positionTitle: 'Contract Specialist 8',
-        positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
-        positionLocation: 'Multiple Locations',
-        organizationName: 'Office of Acquisition and Logistics 5',
-        minimumPay: 0,
-        maximumPay: 150000,
-        rateIntervalCode: 'PA',
-        applicationCloseDate: 'September 24, 2023'
-      }
-    ]
+  const headers = {
+    Accept: 'application/json',
+    'Content-Type': 'application/json'
   };
+  global.fetch = jest.fn(() =>
+    Promise.resolve({
+      json: () => Promise.resolve({})
+    })
+  ) as jest.Mock;
 
   it('renders Jobs component', () => {
```

Comment:
Is there a difference between this test and the one below "shows Jobs details"? can we combine them?

Reply:
updated the test